### PR TITLE
Added travis and coveralls badges to README

### PR DESCRIPTION
**Pull request by majames**
🎫

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-  - "7"
-
+  - 7
 cache:
   yarn: true
-
-install:
-  - yarn
+script:
+  - yarn run test -- --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```
Comment (zzarcon) [inline .travis.yml]: Can we wrap this into a package script and just ```yarn coverage``` later?

**Review by zzarcon — CHANGES_REQUESTED** [review]

**Comment by zzarcon** [comment]
I think the coveralls bot is too noisy, an we avoid somehow the comments on the PRs? what do you think?

**Comment by majames** [comment]
disabled 💬 🙉 ⛔️

File: .travis.yml (hunk 1 above)
Comment (majames) [reply .travis.yml]: The command will only work when run within Travis so I didn't want to add it as a script. I'll add a `yarn coverage` command that makes sense for devs to run locally doe.

**Review by zzarcon — APPROVED** [review]

**Pull request merged**